Source/santad/EventProviders/SNTEndpointSecurityManager.mm

```diff
@@ -77,6 +77,14 @@ - (void)establishClient API_AVAILABLE(macos(10.15)) {
 
     es_client_t *client = NULL;
     es_new_client_result_t ret = es_new_client(&client, ^(es_client_t *c, const es_message_t *m) {
+      // If enabled, skip any action generated from another endpoint security client.
+      if (config.skipOtherEndpointSecurityClients && m->process->is_es_client) {
```

Comment:
Reverse these conditions - I think `is_es_client` is going to be very rare and is a fairly simple field access vs a property get (with all the KVO goodness that comes with it).

Reply:
Done